### DISCUSSION TITLE
Feature request: global comment search

### DISCUSSION THREAD

**Discussion by ados8** [docmost/docmost#1625] [Ideas]
Comments are added by users but there's no way to know they're there unless you are on the document page.
Like my suggested global search it would be great to have a comment list for each space and and a global comment list on home page.
It would need to show a new unread comment you haven't see and allow instant jump to that document.
Hoping this can be a new feature.

**Comment by sy772** [comment]
Also when someone added a comment it can be notification if it is possible.
